addons/point_of_sale/models/pos_config.py

```diff
@@ -804,6 +820,23 @@ def open_ui(self):
         self._check_company_has_fiscal_country()
         return self._action_to_open_ui()
 
+    def close_session_snoozes(self):
+        """
+        Close all snoozes that don't have an end_time when the session is closed
+        """
+        now = Datetime.now()
+
+        snoozes = self.pos_snooze_ids.search([
+            ('end_time', '=', False),
+        ])
+
+        if snoozes:
+            snoozes.write({
+                'end_time': now,
+            })
+
+        return snoozes
```

Comment:
I think keeping snooze is a bit useless and clutters up the database, what do you think about unlink them ?